src/envoy/utils/utils.cc

```diff
@@ -26,6 +26,12 @@ namespace {
 
 const std::string kSPIFFEPrefix("spiffe://");
 
+// Per-host opaque data field
+const std::string kPerHostMixer("mixer");
+
+// Attribute field for per-host data override
+const std::string kMetadataDestinationUID("destination.uid");
```

Comment:
Please change it to "uid".

Add comment as
// This is same as defining following proto
//  message PerHostData {
//        string   uid = 1;
// }